testing/python/fixtures.py

```diff
@@ -4536,3 +4536,41 @@ def test_fixt(custom):
     result.assert_outcomes(errors=1)
     result.stdout.fnmatch_lines([expected])
     assert result.ret == ExitCode.TESTS_FAILED
+
+
+@pytest.mark.xfail(
```

Comment:
We usually don't add xfail skipped tests, perhaps open an issue with the test case instead? (Unless it's fixed by one of the follow up PRs, then can add it there)